src/_lib/sass/src/_variables.scss

```diff
@@ -62,6 +63,17 @@ $color-settings-maps: (
     $coolgray-color-settings
 );
 
+//== Levels colors
+//
+$level0: $levels-base; // #121417
+$level1: lighten($levels-base, 3); // #191b20
+$level2: lighten($levels-base, 6); // #1F2328
+$level3: lighten($levels-base, 9); // #262A31
+$level4: lighten($levels-base, 12); // #2D3239
+$level5: lighten($levels-base, 15); // #343942
+// Used in preview app
+$levels-map: (0: $level0, 1: $level1, 2: $level2, 3: $level3, 4: $level4, 5: $level5);
```

Comment:
If you need this I'd move it to the demo component's stylesheet. I don't think it makes sense to add variables here that are solely used by the demo app